From 26728aa63539fc3c58d2de8a3e066fc4554a55fe Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Mon, 25 Aug 2008 11:07:54 +0800
Subject: [PATCH] ENGR00088936 MX51: Enable VPU and USB modules in default config file

Enable VPU and USB modules in imx51_3stack_defconfig file.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 arch/arm/configs/imx51_3stack_defconfig |   18 +++++++++++++++---
 1 files changed, 15 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/imx51_3stack_defconfig b/arch/arm/configs/imx51_3stack_defconfig
index d46b799..7e1e59d 100644
--- a/arch/arm/configs/imx51_3stack_defconfig
+++ b/arch/arm/configs/imx51_3stack_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.24
-# Wed Aug 20 11:18:20 2008
+# Mon Aug 25 10:58:31 2008
 #
 CONFIG_ARM=y
 CONFIG_SYS_SUPPORTS_APM_EMULATION=y
@@ -651,6 +651,7 @@ CONFIG_TOUCHSCREEN_MXC=y
 # CONFIG_TOUCHSCREEN_TOUCHRIGHT is not set
 # CONFIG_TOUCHSCREEN_TOUCHWIN is not set
 # CONFIG_TOUCHSCREEN_UCB1400 is not set
+# CONFIG_TOUCHSCREEN_USB_COMPOSITE is not set
 # CONFIG_INPUT_MISC is not set
 
 #
@@ -948,6 +949,14 @@ CONFIG_SND_SOC_WM8903=y
 # CONFIG_SOUND_PRIME is not set
 # CONFIG_HID_SUPPORT is not set
 CONFIG_USB_SUPPORT=y
+CONFIG_USB_ARCH_HAS_HCD=y
+# CONFIG_USB_ARCH_HAS_OHCI is not set
+CONFIG_USB_ARCH_HAS_EHCI=y
+# CONFIG_USB is not set
+
+#
+# NOTE: USB_STORAGE enables SCSI, and 'SCSI disk support'
+#
 
 #
 # USB Gadget Support
@@ -996,8 +1005,6 @@ CONFIG_USB_G_SERIAL=m
 #
 # 
 #
-CONFIG_NEW_LEDS=y
-CONFIG_LEDS_CLASS=y
 CONFIG_MMC=y
 # CONFIG_MMC_DEBUG is not set
 # CONFIG_MMC_UNSAFE_RESUME is not set
@@ -1009,12 +1016,15 @@ CONFIG_MMC_BLOCK=y
 CONFIG_MMC_BLOCK_BOUNCE=y
 # CONFIG_SDIO_UART is not set
 
+#
 # MMC/SD Host Controller Drivers
 #
 # CONFIG_MMC_SPI is not set
 # CONFIG_MMC_MXC is not set
 CONFIG_MMC_IMX_ESDHCI=m
 # CONFIG_MMC_IMX_ESDHCI_PIO_MODE is not set
+CONFIG_NEW_LEDS=y
+CONFIG_LEDS_CLASS=y
 
 #
 # LED drivers
@@ -1087,6 +1097,8 @@ CONFIG_MXC_MC13892_POWER=y
 #
 # MXC VPU(Video Processing Unit) support
 #
+CONFIG_MXC_VPU=y
+# CONFIG_MXC_VPU_DEBUG is not set
 
 #
 # MXC Asynchronous Sample Rate Converter support
-- 
1.5.4.4

